src/ArrayOfSets.hpp

```diff
@@ -282,23 +264,15 @@ class ArrayOfSets : protected ArrayOfSetsView< T, INDEX_TYPE >
 
   /**
    * @brief Set the number of sets.
-   * @param [in] numSets the new number of sets.
-   * @note We need this method in addition to the following resize method because of SFINAE requirements.
-   */
-  void resize( INDEX_TYPE const numSets ) LVARRAY_RESTRICT_THIS
-  { ParentClass::resize( numSets, 0 ); }
-
-  /**
-   * @brief Set the number of sets.
-   * @param [in] newSize the new number of sets.
-   * @param [in] defaultSetCapacity the default capacity for each new array.
+   * @param newSize the new number of sets.
+   * @param defaultSetCapacity the default capacity for each new array.
    */
-  void resize( INDEX_TYPE const numSets, INDEX_TYPE const defaultSetCapacity ) LVARRAY_RESTRICT_THIS
-  { ParentClass::resize( numSets, defaultSetCapacity ); }
+  void resize( INDEX_TYPE const newSize, INDEX_TYPE const defaultSetCapacity=0 ) LVARRAY_RESTRICT_THIS
+  { ParentClass::resize( newSize, defaultSetCapacity ); }
```

Comment:
```suggestion
  void resize( INDEX_TYPE const numSubSets, INDEX_TYPE const defaultSetCapacity=0 ) LVARRAY_RESTRICT_THIS
  { ParentClass::resize( numSubSets, defaultSetCapacity ); }
```

Reply:
Who that suggestion stuff is cool. Done.